presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -23,5 +23,7 @@ data class EventScreenState(
     val isStartDateTime: Boolean = false,
     val shouldOpenDropdown: Boolean = false,
     val alarmReminderItem: AlarmReminderItem = AlarmReminderItem.TEN_MINUTES,
-    val listOfAttendee: List<Attendee> = listOf()
+    val listOfAttendee: List<Attendee> = listOf(),
```

Comment:
Again
No need to add `listOf` on a list variable
Just `attendees` is more than enough

Reply:
@MKiperszmid updated